src/components/partner/dto/update-partner.dto.ts

```diff
@@ -41,6 +41,10 @@ export abstract class UpdatePartner {
 
   @IdField({ nullable: true })
   readonly languageOfWiderCommunicationId?: IdOf<Language> | null;
+
+  @Field(() => [Language], { name: 'languagesOfConsulting', nullable: true })
+  @Transform(({ value }) => uniq(value))
```

Comment:
Otherwise the an `undefined` will get converted to `[]` which means clear all instead of ignore with no changes.
```suggestion
  @Transform(({ value }) => value ? uniq(value) : undefined)
```